### PR TITLE
m_Do_main OK

### PR DESCRIPTION
**Pull request by TakaRikka**

### CODE REVIEW

File: src/m_Do/m_Do_main.cpp [hunk 1]
```diff
@@ -1,72 +1,490 @@
...
+
+    if (mMaxTotalUsedSize < totalUsedSize)
+        mMaxTotalUsedSize = totalUsedSize;
+
+    if (mMaxTotalFreeSize > freeSize)
+        mMaxTotalFreeSize = freeSize;
 }
 
+// unused data?
+static u32 lit_2100[] = {0x3F800000, 0x3F800000, 0x3F800000, 0x3F800000, 0x3F800000, 0x3F800000};
```
Comment (magcius) [inline src/m_Do/m_Do_main.cpp:54]: mind making these floats?

File: configure.py [hunk 2]
```diff
@@ -313,6 +313,7 @@ def ActorRel(rel_name, status):
             Matching   ("f_pc/f_pc_stdcreate_req.cpp"),
 
             # machine
+            NonMatching("m_Do/m_Do_main.cpp"),
```
Comment (magcius) [inline configure.py:316]: still nonmatching?

**Review by magcius — APPROVED** [review]

File: src/m_Do/m_Do_main.cpp (hunk 1 above)
Comment (TakaRikka) [reply src/m_Do/m_Do_main.cpp:54]: didn't seem to work correctly when using floats

File: configure.py (hunk 2 above)
Comment (TakaRikka) [reply configure.py:316]: yeah there's some weird linking stuff going on I'm unsure about. everything looks matching as far as I can tell though

**Pull request merged**